code/modules/clothing/head/jobs.dm

```diff
@@ -117,6 +117,10 @@
 	else
 		to_chat(user, span_warning("You just took a candy corn! You should wait a couple minutes, lest you burn through your stash."))
 
+//Now if detective wants to get a noir fedora it will be too protected.
```

Comment:
Comment is unclear - what does `it will be too protected.` mean?

Reply:
Before it detective had basic gray fedora in their wardrobe which didn't had protection, and I added a second gray fedora but with protection (For detectives only)